packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -0,0 +1,740 @@
+import {
+  MessageManager,
+  PersonalMessageManager,
+  TypedMessageManager,
+  AbstractMessage,
+  OriginalRequest,
+} from '@metamask/message-manager';
+import { ORIGIN_METAMASK } from '@metamask/controller-utils';
+import {
+  SignatureController,
+  SignatureControllerMessenger,
+  SignatureControllerOptions,
+} from './SignatureController';
+
+jest.mock('@metamask/message-manager', () => ({
+  MessageManager: jest.fn(),
+  PersonalMessageManager: jest.fn(),
+  TypedMessageManager: jest.fn(),
+}));
+
+jest.mock('@metamask/controller-utils', () => {
+  const actual = jest.requireActual('@metamask/controller-utils');
+  return { ...actual, detectSIWE: jest.fn() };
+});
+
+// eslint-disable-next-line jest/prefer-spy-on
+console.info = jest.fn();
```

Comment:
This will change the behavior of `console.info` across all tests. Is that what we want? I think we ought to follow what the ESLint rule says and use `spyOn`.

Reply:
Done.